source/Halibut.Tests/HalibutTimeoutsAndLimitsForTestsBuilder.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Halibut.Diagnostics;
+
+namespace Halibut.Tests
+{
+    public class HalibutTimeoutsAndLimitsForTestsBuilder
```

Comment:
Nit: Just a thought. The "ForTests" part of the name, is that required? One could say all test builds are for tests.

But I guess it's here to hint to the consumer that it is radically different from regular limits.

Happy to keep it as is, but this thought came to mind and I just wanted it out there somewhere 😁